crates/node/engine/src/state/builder.rs

```diff
@@ -104,57 +124,22 @@ impl EngineStateBuilder {
     /// Builds the [EngineState], fetching missing block info if necessary.
     pub async fn build(self) -> Result<EngineState, EngineStateBuilderError> {
         let mut builder = self;
+        debug!(target: "engine", "Building engine state");
```

Comment:
Maybe add some more info to these logs, with the state that was fetched?

Reply:
Yea good idea